### PR TITLE
Add Morph and Alternate Form to Basic Set

### PR DESCRIPTION
**Pull request by Legendsmith**
Full advantages, with modifiers from Basic Set and Powers included.

### CODE REVIEW

File: Library/Basic Set/Basic Set Advantages.adq [hunk 1]
```diff
@@ -18206,4 +18206,137 @@
...
+			<affects>total</affects>
+			<reference>B84</reference>
+		</modifier>
+		<modifier version="1" enabled="no">
+			<name>Absorptive Change</name>
+			<cost type="percentage">5</cost>
+			<levels>1</levels>
+			<affects>total</affects>
+			<reference>P75</reference>
+			<notes>@Level of Absorbptive Change. 1 None, 2, Light, 3, Medium, 4, Heavy, 5, Extra Heavy@</notes>
```
Comment (richardwilkes) [inline Library/Basic Set/Basic Set Advantages.adq:18225]: I believe "Absorbptive" is a misspelling.

**Comment by richardwilkes** [comment]
Were Morph and Alternate Form not already present in the lists?

**Comment by Legendsmith** [comment]
They weren't in the lists, no.

File: Library/Basic Set/Basic Set Advantages.adq (hunk 1 above)
Comment (Legendsmith) [reply Library/Basic Set/Basic Set Advantages.adq]: Fixed, thanks

**Comment by richardwilkes** [comment]
Thanks!

**Pull request merged**